pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter.go

```diff
@@ -169,10 +169,9 @@ func ValidateVirtualMachineInstanceSpec(field *k8sfield.Path, spec *v1.VirtualMa
 	netValidator := netadmitter.NewValidator(field, spec, config)
 	causes = append(causes, netValidator.Validate()...)
 
-	bootOrderMap, newCauses := validateBootOrder(field, spec, volumeNameMap)
```

Comment:
please explain what do you mean by "Centralize all boot-order validations"

Reply:
The validations for the boot order have been scattered in two places.
Disk ones and network ones.

Now they are called from a single function: `validateBootOrder`.

I thought this is clear from the code change, but if you think I should add some text to the commit let me know.

Reply:
no need get it now